pkg/minikube/bootstrapper/bsutil/kubelet_test.go

```diff
@@ -99,15 +102,15 @@ ExecStart=/var/lib/minikube/binaries/v1.17.2/kubelet --authorization-mode=Webhoo
 					},
 				},
 			},
-			expected: `[Unit]
+			expected: fmt.Sprintf(`[Unit]
```

Comment:
Since this is effectively a change detector test, please put in the raw expected content rather than manipulating it with fmt.Sprintf. 

Otherwise it's too difficult for readers to understand what really is expected.

Reply:
done